crates/ty_test/src/lib.rs

```diff
@@ -86,22 +86,34 @@ pub fn run(
                     EmbeddedFileSourceMap::new(&md_index, test_failures.backtick_offsets);
 
                 for (relative_line_number, failures) in test_failures.by_line.iter() {
-                    let absolute_line_number =
-                        source_map.to_absolute_line_number(relative_line_number);
-
-                    for failure in failures {
-                        match output_format {
-                            OutputFormat::Cli => {
-                                let line_info =
-                                    format!("{relative_fixture_path}:{absolute_line_number}")
-                                        .cyan();
-                                println!("  {line_info} {failure}");
-                            }
-                            OutputFormat::GitHub => println!(
-                                "::error file={absolute_fixture_path},line={absolute_line_number}::{failure}"
-                            ),
+                    let (absolute_line_number, line_number_error, emit_failures) = match source_map
+                        .to_absolute_line_number(relative_line_number)
+                    {
+                        Ok(line_number) => (Some(line_number), None, true),
+                        Err(last_line_number) => {
+                            let total = source_map.total_relative_line_count();
+                            let message = format!(
+                                "Relative line {} exceeds total embedded code lines ({total}). \
+                                     This usually means a trailing `# revealed:` without a matching \
```

Comment:
Can you help me understand how `total_relative_line_count` is different from `last_absolute_line_number`?